### DISCUSSION TITLE
6700 is the maximum speed at which the fan spins. It always shows 93% here

### DISCUSSION THREAD

**Discussion by Vaxoie** [seerge/g-helper#465] [General]
Zephyrus G15 GA503RM
[https://imgur.com/a/424C6tQ](url)

What is wrong?
1. The laptop has more maximum speed and does not do this
2. Indicators thinks that the speed may be higher and does not show
3. Or your answer

**Comment by seerge** [comment]
@Vaxoie someone previously mentioned that G15 has 7600max RPM (so it's coded in the app by that request)

You can run in PowerShell as admin 

```
(Get-WmiObject -Namespace Root/WMI -Class AsusAtkWmi_WMNB).DEVS(0x00110013, 1)

```

To turn fan to max , replace 1 with 0 to turn off. And check what is the max rpm will be?
